src/app/app.module.ts

```diff
@@ -33,7 +33,7 @@ import { DetailsSourceComponent } from "./pages/details-source/details-source.co
 import { GlossaryArticleSourceComponent } from "./pages/glossary-article-source/glossary-article-source.component";
 
 export function HttpLoaderFactory(http: HttpClient) {
-  return new TranslateHttpLoader(http);
+  return new TranslateHttpLoader(http, "./assets/i18n/");
```

Comment:
[As far as I can tell](https://github.com/ngx-translate/core/blob/5d04bd849876030b4117bf912a30c1a02a62df19/projects/ngx-translate/http-loader/src/lib/http-loader.ts) this does the same as before but explicitly, right?

Reply:
It does do the same thing, but it is necessary here as we are now serving the application from /new and this requires us to now be explicit about where the files exist (otherwise it wouldn't know where to find them).